src/client/elm/Explorer/View.elm

```diff
@@ -701,14 +700,20 @@ rejectionToItem ctx reason =
             }
 
         AmountTooLarge account amount ->
-            { content = let url = "https://developers.concordium.com/en/testnet4/testnet/references/manage-accounts.html#account-balances"
-                        in [ text "The sending account ", viewAddress ctx <| account, text " has insufficient funds. Note: only funds that are not staked or locked can be transferred (see "
-                           , link [ onClick <| UrlClicked <| Browser.External url ]
-                                                           { url = url
-                                                           , label = el [ Font.underline ] <| text "account balances"
-                                                           }
-                           , text " documentation)."
-                           ]
+            { content =
+                let
+                    url =
+                        "https://developers.concordium.com/en/testnet4/testnet/references/manage-accounts.html#account-balances"
```

Comment:
This link will be obsolete soon so it would be good if either it is removed or parameterized as a build option.